### ISSUE TITLE
[Bug] WPF does not set AutomationProperties so the experience is poor when using Narrator

### ISSUE THREAD

**Issue by peterfoot** [xamarin/Xamarin.Forms#11161]
### Steps to Reproduce

1. Create or run an existing Xamarin Forms application which uses AutomationProperties to set name, help text etc on controls
2. Notice that none of this accessibility info is used and you get only the bare minimum Narrator support
3. Repeat with UWP to hear the difference with the same Xamarin Forms code/XAML

### Expected Behavior
Native Controls should use the Name/HelpText set in the Xamarin Forms app. Behaviour should be as near as possible to consistent across platforms.

### Actual Behavior
On WPF because there is no implementation the Narrator experience is barely useable.

### Basic Information

- Version with issue: All
- Last known good version: n/a
- IDE: n/a
- Platform Target Frameworks:
  - WPF:  All
- Nuget Packages: All
- Affected Devices: All

### Screenshots

n/a

### Reproduction Link

https://github.com/xamarin/xamarin-forms-samples/tree/master/UserInterface/Accessibility
(Needs a WPF head project adding)

### Workaround

n/a
